oddbird/oddcontrast
# Add implementation warning for browsers

![](https://source.unsplash.com/featured/?cute,animal&waaaarnign)

## Description
Fixes #95. 

I ended up with the text "Browser rendering of Oklab for colors outside of the display gamut is incorrect according to the CSS specification." 

The link is to Miriam's [CSS WG](https://github.com/w3c/csswg-drafts/issues/9449) issue.

## Steps to test/reproduce
Change between formats, and see that the warning shows for all unbounded formats.
